src/services/SearchService.js

```diff
@@ -188,11 +188,22 @@ async function addSkillScore(results, query){
       }
       item.skillScore = Math.round(score / query.skillIds.length * 100) / 100
 
-      if(!item.availableForGigs){
+      if(item.isAvailableForGigs === null){
```

Comment:
Are we sure we want to use `===` here and not `==`?

Reply:
No, good catch, thanks.

Reply:
Fixed now